scripts/travis/gtclang-driver-test.sh

```diff
@@ -49,11 +49,13 @@ cmake .. -DCMAKE_CXX_COMPILER="$CXX"
          -DGTCLANG_UNIT_TESTING=ON                                                                                            \
          -DGTCLANG_INTEGRATION_TESTING=ON                                                                                     \
          -DDawn_DIR=${build_dir}/dawn-prefix/src/dawn-build/prefix/dawn/cmake/                                                \
-         -DPYTHON_EXECUTABLE="$PYTHON_DIR"                                                                                    \
+         -DPYTHON_EXECUTABLE="$PYTHON_DIR"                                                                        			  \
       || fatal_error "failed to configure"
 
 make -j2 || fatal_error "failed to build"
 
+bash ${build_dir}/gtclang-prefix/src/gtclang-build/gtclang-tester-no-codegen.sh
```

Comment:
why adding this? isnt part of the ctest?

Reply:
this should not be here... I don't know why it is